dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -231,6 +231,7 @@
 			compatible = "st,stm32-timers";
 			reg = <0x40012c00 0x400>;
 			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00000800>;
+			resets = <&rctl STM32_RESET(APB2, 11U)>;
```

Comment:
dts changes should not be in this commit (drivers: counter: Reset timer using RCC before initialization)

Reply:
Fixed.